notebooks/source/bad_posterior_geometry.ipynb

```diff
@@ -49,7 +49,8 @@
     "from numpyro.diagnostics import summary\n",
     "\n",
     "from numpyro.infer import MCMC, NUTS\n",
-    "assert numpyro.__version__.startswith('0.7.2')\n",
+    "\n",
+    "assert numpyro.__version__.startswith(\"0.7.2\")\n",
```

Comment:
Hi @MarcoGorelli, could you verify if the [update_version](https://github.com/pyro-ppl/numpyro/blob/master/scripts/update_version.py) script is still work? You can modify [this line](https://github.com/pyro-ppl/numpyro/blob/master/numpyro/version.py#L4) and run `python scripts/update_version.py`.

Reply:
Sure - that script isn't run in CI by the way, should it be?

Reply:
Would you be open to using nbQA for this? Then you could already just do
```console
$ nbqa scripts.update_version notebooks/
updating /home/marco/numpyro-dev/notebooks/source/time_series_forecasting.ipynb
```
and have
```diff
$ git diff
diff --git a/notebooks/source/time_series_forecasting.ipynb b/notebooks/source/time_series_forecasting.ipynb
index 4ecb365..01360c5 100644
--- a/notebooks/source/time_series_forecasting.ipynb
+++ b/notebooks/source/time_series_forecasting.ipynb
@@ -48,7 +48,7 @@
     "    set_matplotlib_formats(\"svg\")\n",
     "\n",
     "numpyro.set_host_device_count(4)\n",
-    "assert numpyro.__version__.startswith(\"0.5.0\")"
+    "assert numpyro.__version__.startswith(\"0.7.2\")"
    ]
   },
   {
```

Reply:
> that script isn't run in CI by the way, should it be?

We have been making PRs to bump the versions.

> using nbQA for this

Whoa, `nbqa` is neat! I guess it is somehow overkilled. Could you replace [those lines](https://github.com/pyro-ppl/numpyro/blob/master/scripts/update_version.py#L24-L27) with
```
pattern1 = re.compile('assert numpyro.__version__.startswith\\("[^"]*"\\)')
pattern2 = re.compile('assert numpyro.__version__.startswith\\(\\\\"[^"]*\\\\"\\)')
text1 = f"assert numpyro.__version__.startswith({new_version})"
text2 = text1.replace('"', '\\"')
```

Reply:
Yes, that also works, thanks for the suggestion!

I then tried running `time_series_forecasting.ipynb`, and it still executes fine